### PR TITLE
Feat:fourth modal

### PR DESCRIPTION
**Pull request by Eldow99**
Issue #:
<a href="https://trello.com/b/oudZuA4c/workoutapp">Trello</a>

### PR type
<br>

<p>Please check the type of change your PR introduces:</p>

- [] Bugfix
- [x] Feature
- [] Code style update (formatting, renaming)
- [] Refactoring (no functional changes, no api changes)
- [] Build related changes
- [] Other (please describe):

### Screenshots
<br>

<h4>before:</h4>

<h4>after:</h4>
Снимок экрана 2023-03-03 в 17 10 34
Снимок экрана 2023-03-03 в 17 10 46

### Does this introduce a breaking change?
<br>

- [] yes (please describe):
- [x] no

### Pull request checklist
<br>

<p>Please check if your PR fulfills the following requirements:</p>

- [] Tests for the changes have been added (for bug fixes / features)
- [] Accessibility support has been tested
- [] Responsive layout and functionality have been tested
   - [] desktop and mobile view responsiveness are supported
   - [] no loss of functionality or content when zoomed in to 400%
   - [] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [x] avoids using many levels of prop drilling
- [x] check console on errors and warnings.

<hr>

### CODE REVIEW

File: src/components/InputModal.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+				age: ageD,
+				weight: weightD,
+				height: heightD,
+			}),
+		)
+		setAgeD('')
+		setWeightD('')
+		setHeightD('')
+	}
+	console.log(data)
```
Comment (Adilxa) [inline src/components/InputModal.js:32]: pls remove log

File: src/components/InputModal.scss [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	box-shadow: 0px 0px 3.475px rgba(0, 0, 0, 0.25);
+	backdrop-filter: blur(21.7188px);
+	color: white;
+	border-radius: 13.9px;
+	padding: 30px;
+	width: 695px;
+	height: 592.49px;
+}
+.inputField {
+	// padding: 7px;
```
Comment (Adilxa) [inline src/components/InputModal.scss:19]: remove this one

File: src/components/InputModal.scss [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+	height: 20px;
+	margin: 14px 12px 11px 0;
+	font-size: 16px;
+	background: #222429;
+	color: white;
+	border: 1px solid #727272;
+	text-align: center;
+}
+.inputfield:focus-within {
+	// cursor: pointer;
```
Comment (Adilxa) [inline src/components/InputModal.scss:48]: same

File: src/components/InputModal.scss [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+	width: 357px;
+	margin-top: 15px;
+}
+.inputMainText h1 {
+	font-family: 'Roboto';
+	font-style: normal;
+	font-weight: 500;
+	font-size: 40px;
+	line-height: 47px;
+	/* identical to box height */
```
Comment (Adilxa) [inline src/components/InputModal.scss:113]: same here

File: src/index.js [hunk 5]
```diff
@@ -1,28 +1,26 @@
...
-import { BrowserRouter } from 'react-router-dom';
-
-const root = ReactDOM.createRoot(document.getElementById('root'));
+import React, { Suspense } from 'react'
+import ReactDOM from 'react-dom/client'
+import { Provider } from 'react-redux'
+import { store } from './redux'
+import './index.scss'
+import App from './App'
+// import reportWebVitals from './reportWebVitals'
```
Comment (Adilxa) [inline src/index.js:7]: same

**Review by Adilxa — APPROVED** [review]

File: src/App.js [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
-  return (
-    <div className="App">
-      <Routes>
-      </Routes>
-    </div>
-  );
+	return (
+		<div className='App'>
+			<Routes>
+
```
Comment (russllan) [inline src/App.js:10]: where HomePage in Route?
pls, add it

**Pull request merged**